src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md

```diff
@@ -10,8 +10,8 @@ k6 has packages for Linux, Mac, and Windows. As alternatives, you can also use a
 ### Debian/Ubuntu
 
 ```bash
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
```

Comment:
BTW, we should also update the translated Spanish installation instructions in `src/data/markdown/translated-guides/es/01 Getting started/02 Installation.md`. I can help out with any translation changes if needed. There's no `01 Troubleshooting.md` there, so it should be just a matter of updating the commands.

Reply:
Can you indeed take care of that and add a commit for it? I understand spanish really well, but haven't spoken/written it seriously for like ten years...

Reply:
Of course, I'll appreciate an update to `/es`, too.
----
But, just to note, I've made a lot of edits lately without making the corresponding Spanish translations.
Right now, I'm just trying to update configuration changes (like vusMax getting deprecated).

Even though my written Spanish is OK, it's just OK. To make a translation for each change would more than double the time I'd need to edit anything.

So, if I'm not mistaken, the idea is to make a big translation in a batch sometime.

----
Again, though, I really appreciate all work on the Spanish side, so there's no need to wait, of course.

Reply:
Missing Spanish translations should not block the PRs. We can track it at https://github.com/grafana/k6-docs/issues/452

Reply:
My original comment wasn't about the missing Spanish translation, but just updating the command used in the installation instructions to match, which wouldn't require any translating.

I did that in #646, fixed other inconsistencies and just went and translated the troubleshooting document as well :)

Reply:
Once again, thanks a lot for that @imiric 🙇🏻